bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerMetadata.java

```diff
@@ -79,8 +83,10 @@
     private LedgerMetadataFormat.DigestType digestType;
     private byte[] password;
 
+    private Map<String, byte[]> customData;
```

Comment:
I would prefer not using null for this. It can be just simply a empty map. or change it to Optional<Map<String, byte[]>>